projects/dynamicforms/package.json

```diff
@@ -1,12 +1,19 @@
 {
-  "name": "@fsa/dynamicforms",
+  "name": "@spartacus/dynamicforms",
   "description": "Library for creating dynamic forms, designed to work with Spartacus application",
-  "version": "1.0.0",
+  "version": "0.0.0",
```

Comment:
Typo? Or we are not having any version this release?

Reply:
release-it script works in a way that first version is changed (increased), you have possibility to choose, minor, major or patch, and then release is created and pushed, so we should have zero.